feature-libs/customer-ticketing/occ/adapters/occ-customer-ticketing.adapter.ts

```diff
@@ -122,4 +125,80 @@ export class OccCustomerTicketingAdapter implements CustomerTicketingAdapter {
       },
     });
   }
+
+  uploadAttachment(
+    customerId: string,
+    ticketId: string,
+    eventCode: string,
+    file: File
+  ): Observable<unknown> {
```

Comment:
Upload Attachment endpoint returns a TicketEventAttachment entity.  It looks like you don't use it, but do you think it should be added? Or you are reloading the ticket details upon event creation?

https://spartacus-dev0.eastus.cloudapp.azure.com:9002/occ/v2/swagger-ui/index.html#/Tickets/createTicketEventAttachment

Reply:
yes, TicketEventAttachment is not needed as ticket details is reloaded upon event creation.